tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
you need noEmit to prevent typescript from compiling to a distribution file.
Actually. Wouldn't it be possible to avoid completely rollup and everything and just use typescript to export the file?